app/src/main/java/com/example/parkingsystemjava/mvp/contract/ParkingReservationContract.java

```diff
@@ -1,15 +1,24 @@
 package com.example.parkingsystemjava.mvp.contract;
 
+import com.example.parkingsystemjava.entity.Reservation;
+import com.example.parkingsystemjava.listener.ListenerDateTime;
+import java.util.Calendar;
+
 public interface ParkingReservationContract {
 
     interface ReservationActivityPresenter {
-        void showDatePicker();
+        void showDatePicker(ListenerDateTime listenerDateTime);
+        void addReservation(Reservation reservation, int parkingLot);
+        void setEntryExitDate(Calendar entryExitDate);
     }
 
     interface ReservationActivityView {
-        void showEntryDatePicker();
+        void showDatePicker(ListenerDateTime listenerDateTime);
+        void showDateSelected(String date);
+        void showConfirmationMessage(int parkingLot);
     }
 
     interface ReservationActivityModel {
+        void addReservation(Reservation reservation, int parkingLot);
     }
```

Comment:
Pls rename the contracts alligned oof the father interface

Reply:
done